bindings/go/configuration/resolvers/v1/matcher/matcher_test.go

```diff
@@ -0,0 +1,278 @@
+package matcher
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestComponentMatcher(t *testing.T) {
```

Comment:
this is missing quite a few common blob cases such as {} or ** syntax

Reply:
I will look for a glob package that isn't 6yrs old